lib/ajvValidation.js

```diff
@@ -72,7 +72,8 @@ function validateSchema (schema, valueToUse, options = {}) {
     }
 
     // Ignore unresolved variables from mismatch if option is set
-    else if (options.ignoreUnresolvedVariables && isPmVariable(_.get(valueToUse, dataPath))) {
+    else if (options.ignoreUnresolvedVariables &&
+      isPmVariable(dataPath === '' ? valueToUse : _.get(valueToUse, dataPath))) {
```

Comment:
@VShingala you can simply pass valueToUse as the third prop

`_.get(valueToUse, dataPath, valueToUse)`. This will fallback to valueToUse if `_.get(valueToUse, dataPath)` is undefined

Reply:
@abhijitkane Yes, but that will always fallback to `valueToUse` and not just when `dataPath` is empty. In some edge case if `dataPath` is not correct, this makes sure that `undefined` is returned and not `valueToUse`.

Reply:
@VShingala Can you abstract this out into a function? It's used in 4 places in this PR

Reply:
@abhijitkane Have abstracted function in `schemaUtils.js`. But as there is circular dependency have left out use of it in `ajvValidation.js`.